python/testapi/plugins/common/test_circuit_color_by_method.py

```diff
@@ -58,8 +58,11 @@ def test_apply(self) -> None:
         color.apply(self.instance, id)
 
     def _load_circuit(self) -> int:
-        loader = brayns.BbpLoader(
+        morphology = brayns.MorphologyParameters(
             radius_multiplier=10
         )
+        loader = brayns.BbpLoader(
+            morphology=morphology
+        )
```

Comment:
Just personal preference but to spare some lines and match the JSON schema, we can define everything in one shot with
```
loader = brayns.BbpLoader(
    morphology=brayns.MorphologyParameters(
        radius_multiplier=10
    )
)
```